Categories/Functor/Instance/ZeroTruncation.agda

```diff
@@ -0,0 +1,57 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Functor.Instance.ZeroTruncation where
+
+-- 0-trucation of groupoids as a functor from Groupoids to Setoids.
+--
+-- This is the right-adjoint of the inclusion functor from Setoids to
+-- Groupoids (see Categories.Functor.Adjoint.Instance.ZeroTruncation)
+
+-- FIXME: maybe this should go into Categories.Category.Construction
```

Comment:
No, this is the right place.  `Construction` is when there is an input, `Trunc` is just a `Functor` that doesn't depend on any external data.